test/controllers/site_warnings_controller_test.rb

```diff
@@ -6,20 +6,9 @@ class SiteWarningsControllerTest < ActionController::TestCase
     @warning = site_warnings(:one)
   end
 
-  test 'should get index' do
-    get :index
-    assert_response :success
-    assert_not_nil assigns(:warnings)
-  end
-
-  test 'should get new' do
-    get :new
-    assert_response :success
-  end
-
   test 'should create site_warning' do
     assert_difference('SiteWarning.count') do
-      post :create, params: { warning: { expires: @warning.expires, message: @warning.message, type: @warning.type, user_id: @warning.user_id } }
+      post :create, params: { warning: { expires: @warning.expires, message: @warning.message, level: @warning.level, user_id: @warning.user_id } }
```

Comment:
The migration `db/migrate/20160609222659_rename_warning_type_to_level.rb` renamed `Warning#type` to `level`.

https://github.com/nbgallery/nbgallery/blob/cd6116dd11cf417d17d900aaf511a524d9329260/db/migrate/20160609222659_rename_warning_type_to_level.rb#L4